R/type-factor.R

```diff
@@ -188,15 +188,6 @@ vec_cast.ordered.ordered <- function(x, to, ...) {
   ord_cast(x, to, ...)
 }
 
-#' @export
-vec_cast.factor.ordered <- function(x, to, ...) {
```

Comment:
Is it better to remove these altogether, or use `stop_incompatible_type()` as a reminder to ourselves that we have thought about this, and it wasn't just an oversight? It would also keep other packages from trying to add a factor.ordered method themselves

Reply:
I think for consistency it's better to remove them.